cardano_node_tests/tests/test_cli.py

```diff
@@ -393,6 +394,16 @@ def _check_stake_snapshot(  # noqa: C901
                 stake_snapshot = cluster_obj.g_query.get_stake_snapshot(
                     stake_pool_ids=expected_pool_ids
                 )
+
+                # check stake distribution on dbsync,
+                # the stake distribution is extracted from the set snapshot of the ledger.
+                pool_epoch_stake = dbsync_queries.query_epoch_stake(pool_ids[0])
+
+                assert (
+                    stake_snapshot["pools"][pool_epoch_stake.hash.hex()]["stakeSet"]
+                    == pool_epoch_stake.amount
+                ), "The epoch stake distribution on dbsync is not the expected"
```

Comment:
I don't _think_ this is correct English, maybe "The epoch stake distribution in dbsync doesn't match stake snapshot"?